server/server.go

```diff
@@ -295,18 +301,40 @@ func NewServer(ctx context.Context, prof Profile, logger *zap.Logger, loggerLeve
 		return nil, fmt.Errorf("failed to create license service, error: %w", err)
 	}
 
-	s.InitSubscription()
+	s.initSubscription()
+	s.initMetricScheduler(deploymentID)
```

Comment:
Better add it here and follow the existing pattern https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/server/server.go?L154:5

Also we only need to report metrics if it's in release and non-demo mode.

Reply:
Oh, there is an implicit dependency on initSubscription. Let's make this dependency explicit and pass "plan" to initMetricScheduler

Reply:
I think we can just remove the "plan" if we put the `Identify` into `Run`. Updated.